ui/src/pages/childPostnatalCare.jsx

```diff
@@ -0,0 +1,432 @@
+      ...initialValues,
+    },
+    validationSchema: validationSchema,
+    // submit form
+    onSubmit: (values) => {
+      // console.log(values);
+      setPreview(true);
+      setInputData(values);
+    },
+  });
+
+  function prompt(text) {
+    setMessage(text);
+    setOpen(true);
+    setTimeout(() => {
+      setOpen(false);
+    }, 4000);
+    return;
+  }
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  useEffect(() => {
+    let visit = window.localStorage.getItem("currentPatient");
+    if (!visit) {
+      setMessage(
+        "No patient visit has been initiated. To start a visit, Select a patient in the Clients list"
+      );
+      setOpen(true);
+      setTimeout(() => {
+        setOpen(false);
+      }, 4000);
+      return;
+    }
+    setVisit(JSON.parse(visit));
+    return;
+  }, []);
+
+  useEffect(() => {
+    let visit = window.localStorage.getItem("currentPatient");
+    if (!visit) {
+      prompt(
+        "No patient visit not been initiated. To start a visit, Select a patient in the Client list"
+      );
+      return;
+    }
+    setVisit(JSON.parse(visit));
+    return;
+  }, []);
+
+  useEffect(() => {
+    let visit = window.localStorage.getItem("currentPatient") ?? null;
+    visit = JSON.parse(visit) ?? null;
+    if (visit) {
+      getChildPostnatalCareEncounters(visit.id);
+    }
+  }, []);
+
+  let getEncounterObservations = async (encounter) => {
+    setObservations([]);
+    // handleOpen();
+    let observations = await (
+      await FhirApi({ url: `/crud/observations?encounter=${encounter}` })
+    ).data;
+    setObservations(observations.observations);
+    return observations.observations;
+  };
+
+  let getChildPostnatalCareEncounters = async (patientId) => {
+    setLoading(true);
+    let encounters = await (
+      await FhirApi({
+        url: `/crud/encounters?patient=${patientId}&encounterCode=${"CHILD_POSTNATAL_CARE"}`,
+      })
+    ).data;
+    // console.log(encounters);
+    setChildPostnatalCareEncounters(encounters.encounters);
+    setLoading(false);
+    return;
+  };
+  let saveChildPostnatalCare = async (values) => {
+    //get current patient
+    if (!visit) {
+      prompt(
+        "No patient visit has been initiated. To start a visit, Select a patient in the client's list"
+      );
+      return;
+    }
+    let patient = visit.id;
+    try {
+      //create Encounter
+      let encounter = await createEncounter(patient, "CHILD_POSTNATAL_CARE");
+      // console.log(encounter)
+
+      //Create and Post Observations
+      let res = await (
+        await FhirApi({
+          url:`/crud/observations`,
+          method: "POST",
+          data: JSON.stringify({
+            patientId: patient,
+            encounterId: encounter.id,
+            observations: values,
+          }),
+        })
+      ).data;
+      // console.log(res);
+
+      if (res.status === "success") {
+        prompt("Present baby saved successfully");
```

Comment:
Change feedback display to match the name of your form, which is "Baby postnatal care"